### PR TITLE
feat(github): getAccessToken

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

GitHub connector's `getAccessToken`

https://www.notion.so/silverhand/Social-470168a733904b7ba2b1f52dd5b131d0

## Testing

UT

## Reviewers

@logto-io/eng

### CODE REVIEW

File: packages/core/src/connectors/github/index.ts [hunk 1]
```diff
@@ -36,3 +37,25 @@ export const getAuthorizeUri: GetAuthorizeUri = async (redirectUri, state) => {
     scope, // Only support fixed scope for v1.
   })}`;
 };
+
+export const getAccessToken: GetAccessToken = async (code) => {
+  const { clientId, clientSecret } = await getConnectorConfig<GithubConfig>(
```
Comment (simeng-li) [inline packages/core/src/connectors/github/index.ts:42]: nit: 
```
const { clientId: client_id, clientSecret: client_secret} = await getConnectorConfig();
```

**Review by simeng-li — APPROVED** [review]
overall LGTM just wondering for the `getAccessToken` and `getAuthorizationUri` methods, could it be shared by other connectors, with some customized input params?  Looks pretty general.

**Comment by gao-sun** [comment]
@simeng-li auth uri looks general while other socials may don't have the concept of `access_token`. we can explore more and see how it goes.

**Review by gao-sun — APPROVED** [review]
@simeng-li actually @wangsijie defined the type of `getAuthorizationUri` and `getAccessToken` so i think it's good for now

**Pull request merged**